src/Parallel/PhaseControl/PhaseControlTags.hpp

```diff
@@ -84,4 +91,78 @@ using reduction_data = Parallel::ReductionData<Parallel::ReductionDatum<
     tuples::tagged_tuple_from_typelist<TagsAndCombinesPresent>,
     TaggedTupleCombine>>;
 
+namespace OptionTags {
+/// Option tag for the collection of triggers that indicate synchronization
+/// points at which phase changes should be considered, and the associated
+/// `PhaseChange` objects for making the phase change decisions.
+///
+/// \note The nested collection types for this option tag gives the yaml
+/// format the slightly unusual form:
+///
+/// ```
+/// PhaseChangeAndTriggers:
+///   - - Trigger1
+///     - - PhaseChange1
+///       - PhaseChange2
+///     - Trigger2
+///     - - PhaseChange3
+///       - PhaseChange4
+/// ```
```

Comment:
Using a map will give you a nicer format, assuming you don't care about the order the triggers are given in.  (And I think you forgot a "-" on line 107.)